GymBro/Views/RoutineViews/RoutineSelectionVIew.swift

```diff
@@ -8,9 +8,86 @@
 
 import SwiftUI
 
+/// Summary
+///
+/// This is a page show up after clicking "new routine" button in NewRoutineVIew. Customer will able to select their desire routnie and  add a new blank reoutn in this page. By clicking any one of the routine button, it will link to the correspond RoutineExerciseView.
+///
+///
 struct RoutineSelectionVIew: View {
```

Comment:
Recommend documentation: 

```swift
/// View after NewRoutineView
///
/// This is a page show up after clicking "new routine" button in NewRoutineVIew. Customer will able to select their desire routnie and  add a new blank reoutn in this page. By clicking any one of the routine button, it will link to the correspond RoutineExerciseView.
struct RoutineSelectionVIew: View {...}
```